notebook/static-src/edit/js/main.js

```diff
@@ -1,33 +1,22 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
+"use strict";
+
+require('base/js/globals').then(function() {
+    
+    var IPython = require('base/js/namespace');
```

Comment:
The globals -> namespace require is quite scary to me here, but we have to deal with this in sequence for the current notebook anyway.